lib/dynflow/world.rb

```diff
@@ -228,6 +228,7 @@ def event(execution_plan_id, step_id, event, done = Concurrent::Promises.resolva
     end
 
     def plan_event(execution_plan_id, step_id, event, time, accepted = Concurrent::Promises.resolvable_future)
+      time = Time.now + time if time.is_a?(Numeric)
```

Comment:
Numerical `time` param is processed on orchestrator side now.
:warning: this could delay the events for the communication error! Is that acceptable?

Reply:
It shouldn't be an issue when things are working smoothly, but I'd be worried what would happen once the load got high enough. What was the reason for this change?

Reply:
The reason is just tests - with testing clock, we can't have real time, but probably we can think of better solution :thinking:

Reply:
I'd like to keep the option to send times, if possible. Is it doable in the time we have left?

Reply:
Ok, I did it through the clock. It now provides `current_time` even in production. Providing wrapper arround Time.now